modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/IgniteCacheDumpSelfTest.java

```diff
@@ -146,6 +152,43 @@ public void testCacheDump() throws Exception {
         }
     }
 
+    /** */
+    @Test
+    public void testZippedCacheDump() throws Exception {
```

Comment:
I suppose we should run all tests for compressed dump. Let's make this test parameterized